frontend/workflows-lib/tests/components/TasksFlow.test.tsx

```diff
@@ -1,30 +1,26 @@
 import { render } from "@testing-library/react";
+import "@testing-library/jest-dom";
```

Comment:
still needed?

Reply:
Yeah, weirdly `jest-dom` isn't actually `jest` specific and is recommended as the to test the dom by `testling-library`, see: https://testing-library.com/docs/svelte-testing-library/setup/#vitest